package/Dockerfile

```
@@ -53,8 +53,8 @@ ENV CATTLE_SYSTEM_AGENT_UPGRADE_IMAGE rancher/system-agent:${CATTLE_SYSTEM_AGENT
 ENV CATTLE_SYSTEM_UPGRADE_CONTROLLER_CHART_VERSION 100.0.3+up0.3.3
 
 # System charts minimal version
-ENV CATTLE_FLEET_MIN_VERSION=100.1.0+up0.4.0
-ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=1.0.6+up0.2.7
+ENV CATTLE_FLEET_MIN_VERSION=100.2.0+up0.5.0
```

Update the version following https://github.com/rancher/charts/pull/2346/files#diff-1d00b840e705f8e431bd805362d77aeda19db7c216857eac7bf94c28a0478d13R15

```suggestion
ENV CATTLE_FLEET_MIN_VERSION=100.2.0+up0.5.1
```